installer/helm/chart/volcano/config/volcano-scheduler-ci.conf

```diff
@@ -9,3 +9,5 @@ tiers:
   - name: predicates
   - name: proportion
   - name: nodeorder
+action-arguments:
```

Comment:
please refer to plugins implementation.

Reply:
@k82cn , plug in has lot of other parameters and plugin object creation happens on every opensession. current Reg of plug in and action handling is different. so i felt with minimal code changes this would be fine. ok , will change as plug in implementation